### ISSUE TITLE
Cannot import Postman collection

### ISSUE THREAD

**Issue by BigNerd** [usebruno/bruno#1805]
When trying to import the attached postman collection I get "Import collection failed". No idea why it failed. I am attaching the said collection, maybe you can provide advice or a fix.
[holiday-calendar-api.json](https://github.com/usebruno/bruno/files/14598810/holiday-calendar-api.json)

I am using Bruno v1.11.0 on macOS Sonoma 14.3. I installed Bruno with `brew install Bruno`.
I used the menu action 'Import Collection'/'Postman Collection' and pointed the import to the Postman file attached.

**Comment by bpoulaindev** [comment]
Hi @BigNerd did you export it through Postman directly ? In version 2.1 ? I ran into the same issue as you described with this file. However, converting it to a v2.1 structure with chatGPT made it compatible with Bruno.
[Calendrier vacances API.json](https://github.com/usebruno/bruno/files/14603960/Calendrier.vacances.API.json)

**Comment by BigNerd** [comment]
Fantastic, thanks a lot, the converted version works for me, too. To answer your question, I created the original file that Bruno failed to import by using Postman 10.24.0 lightweight client / Settings... / Data / Export Sctatch Pad Data on macOS Sonoma 14.3. That Postman version was the one that I got after accepting the upgrade proposal from the Postman version I had been using up to then and it gave me the new version that wanted to force me into their cloud which I refused.

Did you use a particular prompt and a specific chatGPT version for the file conversion? I am asking because I have a few more collections that I probably need to convert like you did, too.
